packages/compute-baseline/src/baseline/index.test.ts

```diff
@@ -74,3 +75,37 @@ describe("computeBaseline", function () {
     assert.equal(result.baseline_high_date, "2018-01-29"); // 30 months later
   });
 });
+
+describe("keystoneDateToStatus()", function () {
+  it('returns `"low"` for recent dates', function () {
+    const status = keystoneDateToStatus(
+      Temporal.Now.plainDateISO().subtract({ days: 7 }),
+    );
+    assert.equal(status.baseline, "low");
+    assert.equal(typeof status.baseline_low_date, "string");
+    assert.equal(status.baseline_high_date, null);
+  });
+
+  it('returns `"high"` for long past dates', function () {
```

Comment:
Since these strings will never be treated as markdown, I like this better:

```suggestion
  it('returns "high" for long past dates', function () {
```